idp-service/src/main/java/io/mosip/idp/services/AuthorizationServiceImpl.java

```diff
@@ -64,21 +65,21 @@ public class AuthorizationServiceImpl implements io.mosip.idp.core.spi.Authoriza
 
 
     @Override
-    public OAuthDetailResponse getOauthDetails(String nonce, OAuthDetailRequest oauthDetailReqDto) throws IdPException {
+    public OAuthDetailResponse getOauthDetails(OAuthDetailRequest oauthDetailReqDto) throws IdPException {
         Optional<ClientDetail> result = clientDetailRepository.findByIdAndStatus(oauthDetailReqDto.getClientId(),
                 Constants.CLIENT_ACTIVE_STATUS);
         if(!result.isPresent())
             throw new InvalidClientException();
 
-        log.info("Valid client id found, proceeding to validate redirect URI");
+        log.info("nonce : {} Valid client id found, proceeding to validate redirect URI", oauthDetailReqDto.getNonce());
```

Comment:
Since nonce is optional as per the spec, this can come as null, will our code break anywhere on that case ?

Reply:
Nope it will not break, it will print as below
{"@timestamp":"2022-09-14T22:37:31.900+05:30","@version":"1","message":"nonce : null Valid client id found, proceeding to validate redirect URI","logger_name":"io.mosip.idp.services.AuthorizationServiceImpl","thread_name":"http-nio-8088-exec-2","level":"INFO","level_value":20000,"appName":"idp-service","traceId":"ffd9a375a858737d","spanId":"ffd9a375a858737d","spanExportable":"false","X-Span-Export":"false","X-B3-SpanId":"ffd9a375a858737d","X-B3-TraceId":"ffd9a375a858737d"}